war/src/main/java/io/lumeer/engine/rest/dao/Attribute.java

```diff
@@ -19,50 +19,34 @@
  */
 package io.lumeer.engine.rest.dao;
 
-import io.lumeer.engine.api.LumeerConst;
+import io.lumeer.engine.api.LumeerConst.Collection;
 import io.lumeer.engine.api.data.DataDocument;
 
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
 /**
  * @author <a href="alica.kacengova@gmail.com">Alica Kačengová</a>
  */
 public class Attribute {
```

Comment:
Let's move both `Attribute` and `CollectionMetadata` classes to `io.lumeer.engine.api.dto` package where you can find examples of other similar DTOs.